styles/main.css

```diff
@@ -48,12 +49,56 @@ a {
 }
 
 footer {
-  background: white;
+  background: darkgray;
   height: 64px;
   display: flex;
-  flex-direction: row;
+  position: 0;
+  bottom: 0;
+  width: 100%;
+  flex-direction: row-reverse;
 }
 
 .snicons a {
   object-fit: contain;
+}
```

Comment:
I think we should create different CSS files with the classes we use just in our pages